data/remnant missions.txt

```diff
@@ -1252,3 +1252,232 @@ mission "Remnant: Expanded Horizons Quarg 3"
+				defer
+	on complete
+		conversation
+			`The team waiting for you on <planet> is definitely eager to begin. You barely have the hatch open before they are chivying their camel-pulled wagon up the ramp. They load the broken jump drive surprisingly fast despite their delicate movements.`
+			`	"This is an exciting day, Captain <first>! Countless theories may be proven or sent back for more work based on the testing we can do with this!"`
+			choice
+				`	"Any theories of note?"`
+					goto studying
+				`	"Great! I will find more."`
+			label studying
+			`	He looks thoughtful for a moment, then warbles "As you may know, standard hyperdrives work by generating a fusion reaction and folding it into the fabric of space/time, and these folds naturally fall along the known hyperlanes. I suppose one could view those lanes as the 'crease lines' of the universe. My favorite theory is that jump drives have some additional machinery to guide the fold. That way they can dictate where the fold occurs instead of just aligning with the nearest crease. Given their increase fuel draw, it makes sense that it would take more fuel to generate a fresh fold than to re-use an existing crease."`
+			`	"Oh, before I forget.." he hands you <payment>. "Off to start setting up the tests" he trundles off after the wagon disappearing down a distant corridor.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 4"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	to offer
+		has "Remnant: Broken Jump Drive 3: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`You head over to the Director's desk to see who is in need of the next broken jump drive. The director their smiles at you, looks over several messages on her screen, then clears her throat.`
+			`	"Your finds are very much in demand," she sings, "could you bring this one to a team working on <planet>?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Thank you. We are looking forward to seeing what comes from this."`
+				accept
+			label refuse
+			`	She taps something on her screen. "OK, I've notified them they will have to wait a bit. Let us know when you are ready."`
+				defer
+	on complete
+		conversation
+			`As you approach your designated pad you spot a team of researchers with unloading equipment hustling out of a nearby entrance. Once you have completed your shutdown sequence they quickly set to work unloading the drive. Once complete, they pause for a moment to hand you <payment>. "Thanks for finding this" they sing in unison. "Our labs are eager to begin work on your most recent find, Captain <first>. Our thanks!"`
```

Comment:
"complete" is an odd word choice there, as the subject of the clause is "a team of researchers". Please make it either "once the task is complete" or change the word to "finished" or something else that applies more naturally.
Add a comma after "Thanks for finding this" but inside the quotes.